client/components/NavBar.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import radioButton from '../../server/public/img/radbut1.png';
+import notesButton from '../../server/public/img/notesbutton.png';
+
+export default function NavBar(props) {
+  const { hideMusicWidget, onNotesButtonClick } = props;
```

Comment:
I prefer to use destructuring in the parameters, but that's stylistic and is up to you.